### DISCUSSION TITLE
Welcome to LoRa_APRS_iGate Discussions!

### DISCUSSION THREAD

**Discussion by peterus** [lora-aprs/LoRa_APRS_iGate#101] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself, your hardware and antenna setup.

**Comment by SensorsIot** [comment]
Hi,
we implemented 8 iGates all based on standard TTGO boards near Basel and they work very well. They cover a substantial portion of the area. Some are at home locations, and others at places we already have other equipment. If there is a transmitter (relay) close to the iGate it obviously does not work as well (currently we have no additional filter implemented).
What I am still interested in is the Digipeater function. Unfortunately, I was not able to find documentation and make it work. Maybe somebody can help?

**Reply by micfuc** [reply]
Hi,  we are presently making some experiments with  LoRa APRS using a SH123_esp32_loraprs  derived SW build , designed to run on an own designed modular esp32 based miniplatform; we presently implement both tracker, digipeater, iGate, TNC KISS interface functionalities on a single SW build able to run on several esp32 based platforms ( including the ubiquitos TTGO boards).  Should you be interested in our experiments I attach few posts we made on the subject and a link to a draft copy of the installation manual for our HW/SW.    
 http://www.sarimesh.net/?s=lora 
 http://www.sarimesh.net/wp-content/uploads/2021/08/LoRa_Tracker_manual_vr1_5.pdf   Michele I8FUC

**Comment by TomF8COD** [comment]
Hello Peter,
I have used your code and the one from sh123 on github, and I would like to know if you have plan to implement kiss/AX25 format in your code ? I like to play with these devices, they are easy to put in action. What I like with the sh123 version is the ability to use the esp32 as a simple tnc for aprs droid, and the possibility to send text messages. It would be great to make your code and his interoperable.
Thank for your work, it's really nice to see innovations like this in aprs.
73 de Tom F8COD

The code I'm talking about is this one : https://github.com/sh123/esp32_loraprs

**Comment by stevewa2066** [comment]
@peterus 
I seem to be having a compile error. I"m not a developer at all.. FYI. I got the tracker version to compile and configure just fine. Seems I cannot get the system to have my TTGO V1 board. 


[compile error.txt](https://github.com/lora-aprs/LoRa_APRS_iGate/files/6911861/compile.error.txt)

**Reply by OH2LAK** [reply]
@stevewa2066 Looks like you are missing a library which the compiler is looking for. Have you managed to solve this already?

..
Erik

**Reply by woyteg** [reply]
I habve the same issue

**Comment by ram-projects** [comment]
Hello peterus, 

I just flashed your iGate project to a LoRa32 V2.1. However, I have a problem with the position data. I have entered my GPS position in the source code, but on the aprs.fi page my position is shown somewhere else. I checked in the serial monitor, when the iGate sends a beacon, the iGate sends a different position than the one I entered. The sent one matches the displayed position on the map. What could be the reason for this?

73, Martin DO6RAM

**Reply by peterus** [reply]
Hi Martin!
What format are you using in the json file?
you must use pure decimal format in degrees, not this special aprs/nmea format!

**Comment by ram-projects** [comment]
Hello Peter, 

you are right, as soon as you use the right file format, it works ;). thanks for the tip.

**Comment by px5e9142** [comment]
Hi,

I have entered the folowing data inside the file "is-cfg.json", compiled and uploaded to LoRa, but when the LoRa turns on a message in the display says to me that still something wrong in "is-cfg.json" file.

{
	"callsign": "PU5PWM-10",
	"network": {
		"DHCP": true,
		"static": {
			"ip": "192.168.0.100",
			"subnet": "255.255.255.0",
			"gateway": "192.168.0.1",
			"dns1": "192.168.0.1",
			"dns2": "192.168.0.2"
		},
		"hostname": {
			"overwrite": false,
			"name": "PU5PWM-10"
		}
	},
	"wifi": {
		"active": true,
		"AP": [
			{
				"SSID": "casa 42 mesh",
				"password": "0123456789"
			}
		]
	},
	"beacon": {
		"message": "LoRa iGATE & Digi, Info: github.com/lora-aprs/LoRa_APRS_iGate",
		"position": {
			"latitude": -25.31427,
			"longitude": -49.23130
		},
		"use_gps": false,
		"timeout": 15
	},
	"aprs_is": {
		"active": true,
		"passcode": "16810",
		"server": "euro.aprs2.net",
		"port": 14580
	},
	"digi": {
		"active": false,
		"beacon": false
	},
	"lora": {
		"frequency_rx": 433550000,
		"gain_rx": 0,
		"frequency_tx": 433550000,
		"power": 20,
		"spreading_factor": 12,
		"signal_bandwidth": 125000,
		"coding_rate4": 5,
		"tx_enable": false
	},
	"display": {
		"always_on": true,
		"timeout": 10,
		"overwrite_pin": 0,
		"turn180": true
	},
	"ftp": {
		"active": false,
		"user": [
			{
				"name": "ftp",
				"password": "ftp"
			}
		]
	},
	"mqtt": {
		"active": false,
		"server": "",
		"port": 1883,
		"name": "",
		"password": "",
		"topic": "LoraAPRS/Data"
	},
	"syslog": {
		"active": false,
		"server": "",
		"port": 514
	},
	"ntp_server": "pool.ntp.org"
}

**Comment by DL6UM** [comment]
Hallo Peter ...     
ich verwende ein DollaTek LoRa32 V2.1 433 MHz ESP32 LoRa   als I-Gate mit Deiner Software.
Leider stürzt das Modul in größeren Abstände ab. Dann friert der Bildschirm ein und die blaue LED leuchtet zusätzlich zur roten.
Gestern war das wieder, einfach Neustart dann funktioniert es wieder.
Keine Ahnung ob das Problem auch andere haben, ich will Dir das nur mitteilen, da mir die Ursache für das Verhalten unklar ist.

73 Uli DL6UM     IGate  DL6UM-10




  https://www.amazon.de/dp/B07RXSKPBX?psc=1&ref=ppx_yo2ov_dt_b_product_details

**Comment by Andrzej1000** [comment]
Hi.
When sending a beacon, the following appears on the screen: "Path:, Type: Error,"...

What is it about? How to fix this error?

**Comment by LBCCROV** [comment]
I was able to get the I-gate loaded and the Tracker loaded without much problem.  When the boards powered up before flashing, they had the default display, large graphic characters, etc.  Now it looks like the display,SetCursor does not have any effect and the top line only has a scrolling right to left display.
Here is a quick vid of what I see.  
The display on the right is the I-Gate and on left the Tracker.  There is also an observable refresh rate difference between the two displays.
https://www.youtube.com/watch?v=GhL-Li7iN3A

Where might I have gone wrong?
Thanks, Scott KN6F
KN6f-10 is the Igate, KN6F-5 is the tracker.
